src/develop/tiling.c

```diff
@@ -1821,7 +1821,7 @@ static int _default_process_tiling_cl_roi(struct dt_iop_module_t *self, struct d
                      + ((size_t)oroi_good.x - roi_out->x) * out_bpp;
 
       dt_print(DT_DEBUG_OPENCL,
-               "[default_process_tiling_cl_roi] tile (%d, %d) with %d x %d at origin [%d, %d]\n", tx, ty,
+               "[default_process_tiling_cl_roi] tile (%ld, %ld) with %d x %d at origin [%d, %d]\n", tx, ty,
```

Comment:
I used %ld because that is what GCC suggested and I didn't know about %z until just now. I will fix that.